csharp-advanced-topics/VolatileInterlockedLockInCsharp/Tests/Tests.cs

```diff
@@ -0,0 +1,79 @@
+using VolatileInterlockedLockInCsharp;
+
+namespace Tests
```

Comment:
Please use file-scoped namespaces. In that way, our code looks cleaner since there is a bit less indentation in it.

Reply:
Done